docs/deployment/browsers-and-cookies.md

```diff
@@ -0,0 +1,111 @@
+
+You can disable this option in the `config.yml`, just disabling the
+`check_browser_version` option. If that option is null, we don't check
+the browser version.
+
+The minimum browser version is also configurable in a deployment. The default
+value can be modified in the `avConfig.js`, updating the `browserUpdate` field,
+but that requires a bit more of knowledge because it will require to rebuild
+all the agora-gui modules in that deployment:
+
+1. Edit the `/home/agoragui/avConfig.js` file
+2. Run the `/home/agoragui/build.sh` script as `agoragui` user.
+
+### Mobile browsers
+
+The voting interface is responsive, that means that it adapts to the browser
+size and this makes the application usable on mobile phones:
+
+![Voting on smart phones](/img/mobile.png "Voting on smart phones")
+
+### Javascript and old browsers
+
+The web interface requires javascript to work, so make sure that you don't have
+javascript disabled on your browser.
+
+The minimum required version of javascript is the ES6 (ECMAScript 2015). This
+version is supported on most browsers, but it's not supported on Microsoft
+Internet Explorer 11, so this browser is not supported.
+
+In some old browsers, it's possible that the interface works but the encryption
+fails with an error like this:
+
+![Ballot encoding error](/img/old-browser-error1.png "Ballot encoding error")
+
+In any case, this kind of errors only occurs in the not supported browsers.
+
+## 2. Cookies required and allowed
+
+nVotes uses the bare minimum cookies needed for the interface usability. These
+cookies are the needed to login and for security reasons.
+
+There's no tracking code or external cookies, all the cookies used by the
+nVotes interface are used just as an local storage for the interface state, and
+in any case these cookies are sent to any external server, they are just used
+to be able to query the API with the authentication and the corresponding
+parameters.
+
+You can check the cookies present and the use in the following lists by GUI
+module.
+
+### Used in all modules
+
+ * **lang**: Used to store the interface language. It'll be set even without
+   login to keep track of the language selected by the user for the next use.
+   again. These cookies are present until the user logouts.
+
+### Admin
+
+ * **\*authevent\***: A set of cookies used to store the admin interface state.
+   This is used to keep the user loged in if the browser is closed and opened
+   again. These cookies are present until the user logouts.
+
+### Booth
+
+ * **\*authevent\***: A set of cookies used to store the booth interface state.
+   In the booth we just store:
+    * The current election ID
+    * The user email and authentication token
+    * If the user is admin
+   These cookies are removed once the user emits the vote.
+
+### Election
+
+No special cookies are used here.
+
+### Configuration
+
+The login cookies expiration time can be configured to be a fixed period
+instead of session cookies.
```

Comment:
@danigm is it instead of, or is it a session cookie with experiation time?

Reply:
It's a session cookie with expiration time, the configuration just add the `expires` option to the [cookies.put](https://docs.angularjs.org/api/ngCookies/service/$cookies#put)